### PR TITLE
Hide tournament matches if game is not selected.

### PR DESCRIPTION
**Pull request by javieracevedo**

### CODE REVIEW

File: src/components/forms/match/teams-form.vue [hunk 1]
```diff
@@ -51,7 +53,10 @@
                         v-model="winningTeam"
                         :multiselect-props="teamsMultiselectCustomProps"
                         :endpoint="teamsEndpoint"
+                        title="Select match winner team"
                         field="name"
+                        search-field="sda"
```
Comment (javieracevedo) [inline src/components/forms/match/teams-form.vue]:
```suggestion
```

File: src/components/forms/match/teams-form.vue [hunk 2]
```diff
@@ -51,7 +53,10 @@
...
+                        filter-fields="ad"
```
Comment (javieracevedo) [inline src/components/forms/match/teams-form.vue]:
```suggestion
```

File: src/components/forms/match/index.vue [hunk 3]
```diff
@@ -2,7 +2,9 @@
     <div>
         <form class="resource-form" @submit.prevent="submitForm()">
             <general-info-form />
-            <teams-form />
+            <transition name="fade">
+                <teams-form v-if="!isMatchesDisabled" />
```
Comment (SparoHawk) [inline src/components/forms/match/index.vue]:
```suggestion
                <teams-form v-if="match.game !== null" />
```

File: src/components/forms/match/index.vue [hunk 4]
```diff
@@ -44,10 +46,14 @@ export default {
...
-            match: state => state.Match.data
+            match: state => state.Match.data,
+            matchGame: state => state.Match.data.game
         }),
         isEditing() {
             return Boolean(this.$route.params.id);
+        },
+        isMatchesDisabled() {
+            return this.matchGame === null;
         }
```
Comment (SparoHawk) [inline src/components/forms/match/index.vue]: Remove.

File: src/components/forms/match/index.vue [hunk 5]
```diff
@@ -44,10 +46,14 @@ export default {
     computed: {
         ...mapState({
             isLoading: state => state.Application.isLoading,
-            match: state => state.Match.data
+            match: state => state.Match.data,
+            matchGame: state => state.Match.data.game
```
Comment (SparoHawk) [inline src/components/forms/match/index.vue]: Remove.

File: src/components/forms/match/teams-form.vue [hunk 6]
```diff
@@ -61,6 +64,7 @@
     </div>
 </template>
 <script>
+import { mapState } from "vuex";
```
Comment (SparoHawk) [inline src/components/forms/match/teams-form.vue]: Remove.

File: src/components/forms/match/teams-form.vue [hunk 7]
```diff
@@ -76,6 +80,9 @@ export default {
         }
     },
     computed: {
+        ...mapState({
+            matchGame: state => state.Match.data.game
+        }),
```
Comment (SparoHawk) [inline src/components/forms/match/teams-form.vue]: Remove.

**Review by SparoHawk — CHANGES_REQUESTED** [review]

**Review by SparoHawk — APPROVED** [review]

**Pull request merged**